tempesta_fw/http_sess_conf.c

```diff
@@ -159,12 +159,30 @@ tfw_cfgop_cookie_set_options(TfwStickyCookie *sticky, const char *options)
 		T_ERR_NL("http_sess: too long cookie options length.\n");
 		return -EINVAL;
 	}
-	sticky->options.data = sticky->options_str;
-	sticky->options.len = len + 2;
 
-	sticky->options_str[0] = ';';
-	sticky->options_str[1] = ' ';
-	memcpy(&sticky->options_str[2], options, len);
+	sticky->options_str[sticky->options.len + 0] = ';';
+	sticky->options_str[sticky->options.len + 1] = ' ';
+	memcpy(&sticky->options_str[sticky->options.len + 2], options, len);
+
+	sticky->options.len += len + 2;
+
+	return 0;
+}
+
+int
+tfw_cfgop_cookie_set_path(TfwStickyCookie *sticky, const char *path)
+{
+	size_t len = strlen(path);
+	if (len + 2 > STICKY_OPT_MAXLEN) {
+		T_ERR_NL("http_sess: too long cookie path length.\n");
+		return -EINVAL;
+	}
+
+	sticky->options_str[sticky->options.len + 0] = ';';
+	sticky->options_str[sticky->options.len + 1] = ' ';
+	memcpy(&sticky->options_str[sticky->options.len + 2], path, len);
```

Comment:
The path is copied into buffer here. But only attribute value, starting `path=` string is missing.